app/views/bets/_bet.html.erb

```diff
@@ -0,0 +1,7 @@
+<tr>
+  <td class="col-md-2"><%= bet.creator_id %></td>
+  <td class="col-md-2"><%= bet.created_at %></td>
+  <td class="col-md-2"><%= bet.expires_at %></td>
+  <td class="col-md-2"><%= bet.bet_total %></td>
+  <td class="col-md-4"><%= bet.bet %></td>
```

Comment:
It might be out a scope for this pull request but I would recommend renaming this field as it is confusing. You refer to this column as "Bet Description" in the table header so maybe this should be called `description`.